assets/src/hooks/useSwings.ts

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from "react"
+import { fetchSwings } from "../api"
+import { Swing } from "../schedule.d"
+
+const useSwings = (): Swing[] | null => {
```

Comment:
this hook is small and could be folded into `SwingsView`, removing the need to unit-test it separately from `SwingsView`

Reply:
Hmm, I'm kind of of two minds on this one. On the one hand we have other hooks that are in the same modules as components that use them, and this hook is unlikely to be used by other components anytime soon. On the other hand, it's part of the data plumbing rather than helping to implement some piece of behavior on the component, and I actually personally like having that unit tested separately. I'll think about it.